locations/spiders/backhaus_nahrstedt_de.py

```diff
@@ -0,0 +1,39 @@
+from scrapy.linkextractors import LinkExtractor
+from scrapy.spiders import CrawlSpider, Rule
+
+from locations.categories import Categories
+from locations.hours import DAYS_DE, OpeningHours
+from locations.structured_data_spider import StructuredDataSpider
+
+
+class BackhausNahrstedtDESpider(CrawlSpider, StructuredDataSpider):
+    name = "backhaus_nahrstedt_de"
+    item_attributes = {
+        "brand": "Backhaus Nahrstedt",
+        "brand_wikidata": "Q798438",
+        "extras": Categories.SHOP_BAKERY.value,
+    }
+    allowed_domains = ["nahrstedt.de"]
+    start_urls = ["https://nahrstedt.de/standorte"]
+    wanted_types = ["Bakery"]
+    days = DAYS_DE
+    rules = [
+        Rule(
+            LinkExtractor(allow=r"\/standorte\/[\w-]+$"),
+            callback="parse_sd",
+        ),
+    ]
+    no_refs = True  # TODO: I am very confused as to why I have to do this.
```

Comment:
Because they all have `itemid="uid87"`

```suggestion

    def pre_process_data(self, ld_data, **kwargs):
        ld_data["@id"] = None
```